src/perf_tool_backend.ml

```diff
@@ -172,6 +174,49 @@ module Perf_line = struct
             (exn : exn)
             ~perf_output:(line : string)]
   ;;
+
+  let%test_module _ =
+    (module struct
+      open Core
+
+      let%expect_test "c symbol" =
```

Comment:
```suggestion
      let%expect_test "C symbol" =
```

Nit here and below

Reply:
Done. Also changed the C++ test to include a space in the symbol after the `=>`.